addons/mail/static/src/core/common/mail_guest_model.js

```diff
@@ -88,7 +70,7 @@ export class Persona extends Record {
     /** @type {ImStatus} */
     im_status = fields.Attr(null, {
         onUpdate() {
-            if (this.eq(this.store.self) && this.im_status === "offline") {
+            if (this.eq(this.store.self_guest) && this.im_status === "offline") {
```

Comment:
Shouldn't the -1 check be here instead?
We probably don't read the `presenceChannel` if we don't call `updateBusPresence`?

Reply:
It was not done before, but I don't think it will hurt.